app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  validates :email, presence: true, format: { with: /\A[^@\s]+@digital.cabinet-office.gov.uk+\z/ }
+  validates :first_name, presence: true
+  validates :last_name, presence: true
```

Comment:
Would it be worth adding minimum length validation for the password? Apologies if that's included in the Devise validators